LayoutTests/fast/forms/color/display-none-input-color-chooser-shown.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<script src="../../../resources/testharness.js"></script>
+<script src="../../../resources/testharnessreport.js"></script>
+<input id="colorPick" type="color" />
+<label for="colorPick" id="labelPick">Pick a color</label>
+<script>
+test (function() {
+    assert_true(window.eventSender !== null);
+}, "window.eventSender is required for the test to run");
+test (function() {
```

Comment:
nit: remove space after `test`
```suggestion
test(function() {
    assert_true(window.eventSender !== null);
}, "window.eventSender is required for the test to run");
test(function() {
```